newroutes/formsgSiteCreation.ts

```diff
@@ -0,0 +1,154 @@
+import { DecryptedContent } from "@opengovsg/formsg-sdk/dist/types"
+import autoBind from "auto-bind"
+import express, { RequestHandler } from "express"
+
+import logger from "@logger/logger"
+
+import { BadRequestError } from "@errors/BadRequestError"
+import InitializationError from "@errors/InitializationError"
+
+import { getField } from "@utils/formsg-utils"
+
+import { attachFormSGHandler } from "@root/newmiddleware"
+import { mailer } from "@services/identity/MailClient"
+import UsersService from "@services/identity/UsersService"
+import InfraService from "@services/infra/InfraService"
+
+const { SITE_CREATE_FORM_KEY } = process.env
+const REQUESTER_EMAIL_FIELD = "Government E-mail"
+const SITE_NAME_FIELD = "Site Name"
+const REPO_NAME_FIELD = "Repository Name"
+
+export interface FormsgRouterProps {
+  usersService: UsersService
+  infraService: InfraService
+}
+
+export class FormsgRouter {
```

Comment:
I think the router might be doing a bit too much work - ideally we've tried to keep the router layer as mostly for the parsing of request data and the business logic with services. I think we can tackle this in future with a separate PR!

Reply:
Agreed. I put this as an item in #461.